conf/messages

```diff
@@ -88,17 +88,16 @@ bc.business-verification.radio-legend = What type of business is it?
 bc.business-verification.firstNameField = First name
 bc.business-verification.lastNameField = Last name
 bc.business-verification.saUTRField = Self Assessment Unique Taxpayer Reference (UTR)
-bc.business-verification.coUTRField = Corporation Tax Unique Taxpayer Reference (UTR)
+bc.business-verification.coUTRField = What is your Corporation Tax Unique Taxpayer Reference (UTR)?
+bc.business-verification.coUTRFieldpsaUTR = What is your Partnership Self Assessment Unique Taxpayer Reference (UTR)?
+bc.business-verification.coUTRFieldsaUTR = What is your Self Assessment Unique Taxpayer Reference (UTR)?
 bc.business-verification.businessNamefield = Registered company name
 bc.business-verification.partnerNameField = Partnership name
 bc.business-verification.partnerNameHint = This is the name that you registered with HMRC
 bc.business-verification.partnerUTRField = Partnership Self Assessment Unique Taxpayer Reference (UTR)
 bc.business-verification.registered-name.hint = This is the registered name on your incorporation certificate.
-bc.business-verification.utr.hint = It can usually be found in the header of any letter issued by HMRC next to headings such as ‘Tax Reference’, ‘UTR’ or ‘Official Use’.
-bc.business-verification.utr.help.question = What is a UTR?
-bc.business-verification.utr.help.answer.agent = It is issued by HMRC when you register your business or for Self Assessment. Your UTR number is made up of 10 or 13 digits. If it is 13 digits only enter the last 10.
-bc.business-verification.utr.help.answer_1 = It is issued by HMRC when you register your business or for Self Assessment. Your UTR number is made up of 10 or 13 digits. If it is 13 digits only enter the last 10.
-bc.business-verification.utr.help.answer_2 = Your accountant or tax manager would normally have your UTR.
+bc.business-verification.utr.findUTR = You can <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.gov.uk/find-lost-utr-number">find a lost UTR (opens in new tab)</a>
```

Comment:
Could we remove the html tags from the message file please

Reply:
Hi @nadinJ I have updated this and updated the content to match the updates in the UTR pattern.
![Screenshot 2023-11-01 at 14 42 52](https://github.com/hmrc/business-customer-frontend/assets/1692222/4636aefd-9c06-4df0-acbf-2bcac78cd0a0)